webroot/src/store/user/user.actions.ts

```diff
@@ -192,6 +196,21 @@ export default {
         dispatch('sorting/resetStoreSorting', null, { root: true });
         dispatch('reset', null, { root: true });
     },
+    clearAllNonAccessTokens: () => {
+        /* istanbul ignore next */
```

Comment:
What does this do? Code coverage related?

Reply:
Yes that makes that line not count in the code coverage tester we are using.

Reply:
@jlkravitz For more context, we don't have a full mock replacement of sessionStorage / localStorage in our test runner at the moment. We only have a very small number of difficult-to-test cases for that, so we only need a very small number of coverage ignores.

Reply:
Got it. Thanks for the context!